deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,223 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "cincinnati.openshift.io/v1alpha1",
+          "kind": "Cincinnati",
+          "metadata": {
+            "name": "cincinnati-instance"
+          },
+          "spec": {
+            "branch": "master",
+            "gitHubOrg": "openshift",
+            "gitHubRepo": "cincinnati-graph-data",
+            "registry": "quay.io",
+            "replicas": 1,
+            "repository": "openshift-release-dev/ocp-release"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    description: Creates and maintains a Cincinnati instance
+    operatorframework.io/suggested-namespace: cincinnati
```

Comment:
does this deserve an `openshift-cincinnati` namespace?

Reply:
I has that originally, but my community instincts made me change it.  OpenShift seems to be the only consumers of Cincinnati, so it probably make sense to have the openshift prefix.